### DISCUSSION TITLE
Unable to access AquaPlan link

### DISCUSSION THREAD

**Discussion by BerkAmine** [aquacropos/aquacrop#106] [AquaPlan]
Hello, the link to AquaPlan is not working, could please update it? thank you

**Comment by tfoster88** [comment]
Hi - it should be up and running again now for you. Let me know if any further issues

**Comment by BerkAmine** [comment]
Thank you

**Comment by Zhao-Yiming-bilp** [comment]
你好，AquaPlan的链接无法使用，能否更新一下？谢谢
